scripts/lint-test.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+# This script checks that the remark linter runs without issue. The linter
+# executes the configuration file .remarkrc.mjs if there is at least one file to
+# lint, so to save linting time, we write a mock file to the content directory.
+
```

Comment:
will this somehow have a race condition or whatever that could cause a false positive if the linter only sees this one file and says "yeah all your files are good" ?

Reply:
The goal is to execute `.remarkrc.mjs` to catch runtime errors, so it would only need to run on one file. Without running `yarn git-update`, the submodules aren't loaded on a fresh `docs-website` clone, so each `content/[0-9]+.x` directory is empty. I didn't want to load submodules and run the linter on all content files, since a linter error that slipped past CI in `gravitational/teleport` would break the workflow in `gravitational/docs-website`.